packages/forma-36-react-components/src/index.ts

```diff
@@ -10,6 +10,7 @@ export type {
   PaddingProps,
   Spacing,
 } from '@contentful/f36-core';
+// export * from '@contentful/f36-icons';
```

Comment:
Do we want to export from here? Some of the icon names clashes with component names, and I think it might be better to force people to use `import { ... } from '@contentful/f36-icons'`.

Reply:
Yeah, I'd allow importing from '@contentful/f36-icons' directly.

We should probably append `Icon` to each icon component, e.g. `Heading` => `HeadingIcon`